files/pt-br/web/css/css_counter_styles/using_css_counters/index.md

```diff
@@ -98,7 +98,10 @@ Produz este resultado:
 
 ## Veja mais
 
-- [counter-reset](/pt-BR/docs/CSS/counter-reset)
-- [counter-increment](/pt-BR/docs/CSS/counter-increment)
-
-_There is an additional example available at [http://www.mezzoblue.com/archives/20.../counter_intu/](http://www.mezzoblue.com/archives/2006/11/01/counter_intu/). This blog entry was posted on November 01, 2006, but appears to be accurate._
+- {{cssxref("contain")}}
+- {{cssxref("counter-reset")}}
+- {{cssxref("counter-set")}}
+- {{cssxref("counter-increment")}}
+- {{cssxref("@counter-style")}}
+- [CSS counter styles](/pt-BR/docs/Web/CSS/CSS_counter_styles) module
+- [CSS lists and counters](/pt-BR/docs/Web/CSS/CSS_lists) module
```

Comment:
These two links seems to not exist in pt-BR nor en-US version.
What pages should it be?

Reply:
> These two links seems to not exist in pt-BR nor en-US version. What pages should it be?

Did you check [the original file](https://github.com/mdn/content/blob/54aa017b1d631f9b0ef90ef6361574843b158bba/files/en-us/web/css/css_counter_styles/using_css_counters/index.md?plain=1#L314)? en-US put these two links in 'See Also' part.

Reply:
Oh sorry, it was my mistake, I was using the preview URL and forgot that other pages doesn't exist there